### PR TITLE
Add WhatsApp Sandbox support to Conversations API

### PR DESCRIPTION
**Pull request by mariuspot**
GROW-584
DASH-8197

Added a flag to enable access to WhatsApp Sandbox for Conversations API

### CODE REVIEW

File: src/MessageBird/Client.php [hunk 1]
```diff
@@ -160,11 +163,11 @@ class Client
      * @param string            $accessKey
      * @param Common\HttpClient $httpClient
      */
-    public function __construct($accessKey = null, Common\HttpClient $httpClient = null)
+    public function __construct($accessKey = null, Common\HttpClient $httpClient = null, Array $config = [])
```
Comment (rjelierse) [inline src/MessageBird/Client.php]: Could you use `array` instead of `Array`, it's an internal PHP type.

File: src/MessageBird/Client.php [hunk 2]
```diff
@@ -160,11 +163,11 @@ class Client
...
     {
         if ($httpClient === null) {
             $this->ChatAPIHttpClient = new Common\HttpClient(self::CHATAPI_ENDPOINT);
-            $this->ConversationsAPIHttpClient = new Common\HttpClient(self::CONVERSATIONSAPI_ENDPOINT);
+            $this->ConversationsAPIHttpClient = new Common\HttpClient(in_array(self::ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX,$config) ? self::CONVERSATIONSAPI_WHATSAPP_SANDBOX_ENDPOINT : self::CONVERSATIONSAPI_ENDPOINT);
```
Comment (rjelierse) [inline src/MessageBird/Client.php]: Space after the comma in `in_array` is missing

File: src/MessageBird/Client.php (hunk 1 above)
Comment (mariuspot) [reply src/MessageBird/Client.php]: I'll change it to make it consistent

**Comment by mariuspot** [comment]
Will add an example

**Pull request merged**